lib/git_reflow/commands/setup.rb

```diff
@@ -15,11 +15,24 @@
     end
 
     choose do |menu|
-      menu.header = "Available remote Git Server services:"
+      menu.header = "Available remote Git Server services"
       menu.prompt = "Which service would you like to use for this project?  "
 
       menu.choice('GitHub')    { GitReflow::GitServer.connect reflow_options.merge({ provider: 'GitHub', silent: false }) }
       menu.choice('BitBucket (team-owned repos only)') { GitReflow::GitServer.connect reflow_options.merge({ provider: 'BitBucket', silent: false }) }
     end
+
+    choose do |lgtm_menu|
+      lgtm_menu.header = "Approval Options"
+      lgtm_menu.prompt = "Would you rather?  "
+
+      lgtm_menu.choice('Set the minimum number of approvals (leaving blank will require approval from all commenters) ') {  
+        print "How many approvals would you like? " 
+        GitReflow::Config.add "constants.minimum_approvals", gets
+      }
+    end
+
+    GitReflow::Config.add "constants.approval_regex", /lgtm|looks good to me|[:\+1:]|:thumbsup:|:shipit:/i 
```

Comment:
The regex is duplicated.  We need to reference a single value everywhere the default is used.  Please update this to reference `GitReflow::GitServer::PullRequest::APPROVAL_REGEX`